imaging/ml/ml_codelab/breast_density_cloud_ml.ipynb

```diff
@@ -116,7 +100,10 @@
     "project_id = \"MY_PROJECT\" # @param\n",
     "location = \"us-central1\" # @param\n",
     "dataset_id = \"MY_DATASET\" # @param\n",
-    "dicom_store_id = \"MY_DICOM_STORE\" # @param"
+    "dicom_store_id = \"MY_DICOM_STORE\" # @param\n",
+    "\n",
+    "# Input data used by AutoML must be in a bucket with the following format.\n",
```

Comment:
this looks like an automl commented bucket in a the CMLE codelab....

Reply:
reverted